packages/compass-e2e-tests/helpers/compass.js

```diff
@@ -373,9 +408,22 @@ async function afterTests({ keychain, compass }) {
       compass = null;
     }
   } finally {
-    if (keychain) {
-      keychain.reset();
+    keychain.reset();
+  }
+}
+
+async function printLogs(compass) {
+  const { client } = compass;
+  const types = (await client.logTypes()).value;
+  for (const type of types) {
+    const logs = (await client.log(type)).value;
+    const filtered = logs.filter(
+      (log) => !['DEBUG', 'INFO'].includes(log.level)
+    );
+    if (filtered.length === 0) {
+      continue;
     }
+    console.log(`${type} logs:`, filtered);
```

Comment:
We use `debug` in all the other places, it comes with a benefit of being configurable by the env, I would suggest you switch this to using `debug`

Reply:
I just forgot about debug. Will add it.